targets/REV3-WSS/main.cpp

```diff
@@ -0,0 +1,113 @@
+/**
+ * This is a program that senses the speed of both wheels on the bike.
+ * This is the main target for the board.
+ */
+
+#include <EVT/dev/MCUTimer.hpp>
+#include <EVT/io/CANOpenMacros.hpp>
+#include <EVT/io/CANopen.hpp>
+#include <EVT/io/UART.hpp>
+#include <EVT/io/pin.hpp>
+#include <EVT/manager.hpp>
+#include <EVT/utils/log.hpp>
+#include <WSS.hpp>
+
+namespace IO = EVT::core::IO;
+namespace log = EVT::core::log;
+namespace time = EVT::core::time;
+namespace DEV = EVT::core::DEV;
+
+// Create a can interrupt handler
+void canInterrupt(IO::CANMessage& message, void* priv) {
+    EVT::core::types::FixedQueue<CANOPEN_QUEUE_SIZE, IO::CANMessage>* queue =
+        (EVT::core::types::FixedQueue<CANOPEN_QUEUE_SIZE, IO::CANMessage>*) priv;
+
+    if (queue != nullptr)
+        queue->append(message);
```

Comment:
This is just generally good practice
```suggestion
    if (queue != nullptr) {
        queue->append(message);
    }
```